com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -627,23 +671,36 @@ internal int[] rendererIndexList
             }
         }
 
+        /// <summary>
+        /// Specifies if the Camera Depth Texture should be used for this <c>UniversalRenderPipelineAsset</c>.
+        /// </summary>
         public bool supportsCameraDepthTexture
         {
             get { return m_RequireDepthTexture; }
             set { m_RequireDepthTexture = value; }
         }
 
+        /// <summary>
+        /// Specifies if the Camera Opaque Texture should be used for this <c>UniversalRenderPipelineAsset</c>.
+        /// </summary>
         public bool supportsCameraOpaqueTexture
         {
             get { return m_RequireOpaqueTexture; }
             set { m_RequireOpaqueTexture = value; }
         }
 
+        /// <summary>
+        /// Returns the downsampling used for this <c>UniversalRenderPipelineAsset</c>.
```

Comment:
This setting is related to the camera opaque texture above. Maybe we add also a reference to that doc / seealso tag.
```suggestion
        /// Returns the downsampling method used when copying the camera color texture after rendering opaques. 
```